documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -618,144 +618,28 @@ This configuration can be sent via POST to a running Kafka Connect service, whic
 [[mongodb-connector-monitoring]]
 === Monitoring
 
-Kafka, Zookeeper, and Kafka Connect all have built-in support for JMX metrics.
-The MongoDB connector also publishes a number of metrics about the connector's activities that can be monitored through JMX.
-The connector has two types of metrics.
-Snapshot metrics help you monitor the snapshot activity and are available when the connector is performing a snapshot.
-Streaming metrics help you monitor the progress and activity while the connector reads CDC collection changes.
+The {prodname} MongoDB connector has two metric types in addition to the built-in support for JMX metrics that Zookeeper, Kafka, and Kafka Connect have.
+
+* <<snapshot-metrics, snapshot metrics>>; for monitoring the connector when performing snapshots
+* <<streaming-metrics, streaming metrics>>; for monitoring the connector when reading CDC table data
```

Comment:
Re "CDC table data", that's quite specific to SQL Server / Db2 with their actual CDC tables. I'll slightly reword this for the other connectors to match their paradigms/terminology and merge.